dotcom-rendering/docs/youtube/youtube-variations-2021-11-17.md

```diff
@@ -0,0 +1,26 @@
+# YouTube Players on theguardian.com
+
+As of 17th November 2021
+
+**Platform**|**Type***|**Content / Element Type**|**Module**|**SSR**|**CSR**|**Domain from CAPI?**|**Domain****|**Example**
```

Comment:
Maybe use numbers?

```suggestion
**Platform**|**Type<sup>1</sup>**|**Content / Element Type**|**Module**|**SSR**|**CSR**|**Domain from CAPI?**|**Domain<sup>2</sup>**|**Example**
```

Reply:
That's much better thanks 👍

Reply:
Good old HTML amirite?

Reply:
Semantic markup - I wonder if it will ever catch on :)

Reply:
sup’ :')